lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,14 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    punc = "!#$%&'()*+,-./:;<=>?@[\]^_`{|}~1234567890"
+    tokens = "".join(text.split()).lower()
```

Comment:
looks like you do not need to call `join`

Reply:
Done